### PR TITLE
Bed and LA Cases DAG

### PR DESCRIPTION
**Pull request by brendanbailey**
Hey Folks - I created dags that would update both the bed availability and LA cases data. The dags are pretty straightforward as they take a table and just upload it to AGOL without any trasnformations.

Let me know if there are any changes you would like me to make

### CODE REVIEW

**Comment by tiffanychu90** [comment]
@brendanbailey The DAG looks good to me, I only made a couple of edits to README for uniformity.

**Comment by brendanbailey** [comment]
Thanks @tiffanychu90! Should I go ahead and merge?

**Comment by ian-r-rose** [comment]
I'm about to take a quick look

File: dags/public-health/covid19/sync-la-cases-data.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    gis = arcgis.GIS("http://lahub.maps.arcgis.com", arcuser, arcpassword)
+    gis_item = gis.content.get(arcfeatureid)
+    gis_layer_collection = arcgis.features.FeatureLayerCollection.fromitem(gis_item)
+    gis_layer_collection.manager.overwrite(filename)
+
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime.datetime(2020, 4, 27),
```
Comment (ian-r-rose) [inline dags/public-health/covid19/sync-la-cases-data.py:28]: Same as above.

File: dags/public-health/covid19/sync-la-cases-data.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    workbook = kwargs.get("workbook")
+    sheet_name = kwargs.get("sheet_name")
+    get_data(filename, workbook, sheet_name)
+
+    # Updating ArcGis
+    arcconnection = BaseHook.get_connection("arcgis")
+    arcuser = arcconnection.login
+    arcpassword = arcconnection.password
+    arcfeatureid = kwargs.get("arcfeatureid")
+    update_arcgis(arcuser, arcpassword, arcfeatureid, filename)
```
Comment (ian-r-rose) [inline dags/public-health/covid19/sync-la-cases-data.py:58]: Same as above.

**Comment by brendanbailey** [comment]
@ian-r-rose - I set date to today and I remove the temp files after arcgis is updated

Let me know if you need anything else

**Pull request merged**